database/migrations/2019_04_03_112948_create_temporaries_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
change the migration file name. maybe `2019_03_23_800000_create_temporaries_table.php